app/src/main/java/org/sil/hearthis/RecordActivity.java

```diff
@@ -77,63 +86,70 @@ protected void onCreate(Bundle savedInstanceState) {
 //				lineView.setText("Grandroid says 'Hello!'");
 //			else
 			lineView.setText(line.Text);
+			lineView.setTag(i);
 			lineView.setTypeface(mtfl, 0);
-			if (i == 0) {
+			if (i == activeLine)
+			{
 				lineView.setTextColor(getResources().getColor(R.color.activeTextLine));
 			}
+			lineView.setOnClickListener(new OnClickListener()
+			{
+				@Override
+				public void onClick(View view)
+				{
+					((TextView)linesView.getChildAt(activeLine)).setTextColor(getResources().getColor(R.color.labelTextColor));
```

Comment:
We're starting to have a few places that know how the active line is highlighted. I'd like to see a method for setting the active line that encapsulates all the code for updating the display (probably something I should have done myself, sorry.)